src/control/cmd/daos_server/start_test.go

```diff
@@ -181,13 +181,6 @@ func TestStartOptions(t *testing.T) {
 				return cfg
 			},
 		},
-		"XS Helpers (bad)": {
-			argList: []string{"-x", "42"},
-			expCfgFn: func(cfg *server.Configuration) *server.Configuration {
-				cfg.Servers[0].WithHelperStreamCount(2)
-				return cfg
-			},
-		},
```

Comment:
Is this the only change to control plane? Do we need to explicitly configure this 'pool' style helper xstream or it's always managed by io server internally?

Reply:
yes it is internally managed by io server, the go server just pass the parameter to io server.
The removed code is it originally check the input "42" is invalid, but it can be valid after this patch.